<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScreen</class>
 <widget class="QWidget" name="RegisterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Register</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveto_login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Already have an account? Login instead </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_register_username">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_register_password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_register_confirm_password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
